package.json

```diff
@@ -49,10 +49,10 @@
     "sha1": "1.1.1"
   },
   "peerDependencies": {
-    "@linkurious/ogma": "4.5.5"
+    "@linkurious/ogma": "4.5.9"
   },
   "devDependencies": {
-    "@linkurious/ogma": "4.5.5",
+    "@linkurious/ogma": "4.5.9",
```

Comment:
no change on the package-lock?

Reply:
it is not done yet, just opened the PR.
I am using umbrella to create a tilt env